src/chainparams.cpp

```diff
@@ -238,6 +243,11 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nStartTime = 1555372800; // April 16, 2019
         consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nTimeout = 1569974400; // October 2, 2019
 
+        // Deployment of block-final miner commitment transaction.
+        consensus.vDeployments[Consensus::DEPLOYMENT_BLOCKFINAL].bit = 1;
+        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nStartTime = 1199145601; // January 1, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nTimeout = 1230767999; // December 31, 2008
```

Comment:
In ec8fe58bc5e998b68676384327ade069b7a0c321 

Should be `DEPLOYMENT_BLOCKFINAL` not `DEPLOYMENT_LOCKTIME`. Are you sure those dates are reasonable? If you want this to always be on from start, consider instead using `Consensus::BIP9Deployment::ALWAYS_ACTIVE` for `nStartTime` and `Consensus::BIP9Deployment::NO_TIMEOUT` for `nTimeout`.

Reply:
I kept the above comments, but realized in future commits that you already fixed this. Should probably rebase-edit the commit to remove the 2nd-3rd lines entirely here.